src/helpers/webdavHelper.js

```diff
@@ -42,14 +41,23 @@ exports.delete = async function (userId, file) {
 
   const filename = file.split("/")[file.split("/").length - 1];
 
+  /**
+   * custom sleep
+   */
+  const customDelay = function (time) {
+    return new Promise((resolve) => {
+      setTimeout(resolve, time);
+    });
+  };
+
```

Comment:
Put this on a different helper file, not webdavHelper. If there is not a appropriate file you can create new one